include/IECore/Spline.inl

```diff
@@ -372,6 +372,18 @@ inline bool Spline<X,Y>::operator!=( const Spline &rhs ) const
 	return basis!=rhs.basis || points!=rhs.points;
 }
 
+template<typename X, typename Y>
+IECORE_API void murmurHashAppend( IECore::MurmurHash &h, const Spline<X,Y> &data )
+{
+	h.append( data.basis.step );
+	h.append( data.basis.matrix );
```

Comment:
These two are the other way around from the specialisation we have in `SharedDataHolder<Spline>::hash()`. Probably worth matching the old order so that hashes don't change. We should also remove the `SharedDataHolder` specialisation since it will be unnecessary now.

Reply:
Fixed squashed into 467a02b